debian/changelog

```diff
@@ -3,17 +3,23 @@ cloud-init (25.1-0ubuntu1~20.04.1) UNRELEASED; urgency=medium
   * Drop cpicks which are now upstream:
     - d/p/cpick-d75840be-fix-retry-AWS-hotplug-for-async-IMDS-5995
     - d/p/cpick-84806336-chore-Add-feature-flag-for-manual-network-waiting
-  * refresh patches
+  * refresh patches: (patch hunks are now sorted)
```

Comment:
I wonder if it makes sense to keep the original list of non-sort-related refreshes and then have a separate bullet point mentioning that all patches are now sorted. This will probably be a pain to review for SRU, so I'm sure any additional help/pointers would be appreciated.

Reply:
@TheRealFalcon I just updated the changelog. Let me know what you think.